packages/next-server/server/next-server.ts

```diff
@@ -241,7 +241,7 @@ export default class Server {
       {
         match: route('/api/:path*'),
         fn: async (req, res, params, parsedUrl) => {
-          const { pathname } = parsedUrl
+          const { pathname, query } = parsedUrl
```

Comment:
```suggestion
          const { pathname } = parsedUrl
```
Looks like this isn't used here